File: Introduction to Computer Science - CS50x/week6 - Python/pset6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int
while True:
    x = get_int("heigth: ")
    if x < 9 and x > 0:
        break
for i in range(1, x + 1):
    for n in range(x - i):
        print(" ", end="")
    for m in range(i):
        print("#", end="")
    print("")

File: Introduction to Computer Science - CS50x/week1 - C/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int x;
    // Selecting the Hight
    do
    {
        x = get_int("Hight: ");
    }
    while (x < 1 || x > 8);

    for (int n = 1; n <= x; n++)
    {
        // Spaces
        for (int y = x - n; y > 0; y--)
        {
            printf(" ");
        }
        // Bricks
        for (int i = 1; i <= n; i++)
        {
            printf("#");
        }
        printf("\n");
    }
}